content/chainguard/chainguard-images/staying-secure/fedramp-considerations/index.md

```diff
@@ -0,0 +1,110 @@
+<br /> 
+
+This architecture drives significant friction for vendors delivering FIPS compliant workloads for  modern cloud-native applications. First, very few versions of the Linux Kernel are certified. Second, Linux Kernel certification timelines are often long and arduous. 
+Ultimately, this results in a very limited choice of certified runtimes and compatible underlying hardware for developers to build on. In practice, it means that a certified kernel from a given vendor might be over 5 years old. Outdated kernels typically lack support and optimizations for the latest generation of hardware, and are often incompatible with the latest cloud instance types. It also means when sticking to the same vendor, the application runtimes are equally as out of date and vulnerable.
+Chainguard's solution has the FIPS module and the SP 800-90B entropy source co-located in the container image userspace. This eliminates the need for a certified Linux kernel for the majority of workloads and streamlines engineering effort for workload deployments. This is why Chainguard FIPS images now ship with a certified userspace SP 800-90B entropy source, as in this design:
+
+<center><img src="fedramp-considerations-3.png" alt="Diagram labeled Kernel Independent FIPS Images. The diagram contains a box labeled 'New Chainguard FIPS Image' which contains three objects: a box labeled 'Application', a box labeled 'FIPS Module' (which is surrounded by a rectangle labeled 'FIPS Cryptographic Boundary') and a cylinder labeled 'SP 800-90B Entropy Source.' An arrow points from 'Application' to 'FIPS Module,' and another arrow points from 'FIPS Module' to the 'Entropy Source' cylinder." style="width:1050px;"></center>
+<br /> 
+
+This means that the entropy source is now independent of the hardware or cloud environment. Essentially, you can have FIPS on any host OS, kernel, and hardware. You can even have FIPS on managed cloud kubernetes platforms like GKE, EKS, and AKS. Note that this solution has been tested by two NIST labs and [certified with its own CMVP](https://csrc.nist.gov/projects/cryptographic-module-validation-program/entropy-validations/certificate/191).
+
+For more information, please refer to the CVMP entries for Chainguard's Fips Modules and entropy source:
```

Comment:
I don't recall our style guide's direction on this; do we use "please" and other polite words?

Reply:
I don't think there's anything in the style guide specifically about "please" but there's definitely precedent for using it in this context: https://github.com/chainguard-dev/edu/blob/6324a74c8d1dce68aaebd6e261082e8f991d1963/content/open-source/build-tools/apko/overview/index.md?plain=1#L25 (among other examples)

Reply:
I do think we should avoid it and other polite language outside of this context, though.